### DISCUSSION TITLE
TE3 ?

### DISCUSSION THREAD

**Discussion by Greywind299792458** [HEIG-VD-WEB/web-course#41] [Q&A] [answered]
Bonjour @Nortalle ,
Je n'arrive plus à retrouver l'info dans le site du cours: 

Est-ce qu'on aura encore un troisième test et si oui quand et sur quels sujets ? 
J'ai entendu dire qu'il serait ce jeudi mais je n'ai pas pu trouver des infos vérifiant cela.

**Comment by Nortalle** [comment, marked as answer]
Bonjour @Greywind299792458,

Pour donner suite à notre échange de mail, j'ai mis les informations sur : https://heig-vd-web.github.io/web-course/educational-objectives/part-3/

à tout à l'heure
